### PR TITLE
updates to work with new cortipy / REST API config

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
@subutai please review. With these revisions and [cortipy PR#4](https://github.com/numenta/cortipy/pull/4) we can successfully run the fluent prediction demos (childrens_text and foxeats, via tools/read.py).

### CODE REVIEW

File: fluent/corti_py.py [hunk 1]
```diff
@@ -18,30 +18,39 @@
...
+        "set it with $ export REST_API_KEY=api_key\n"
+        "You can retrieve a key by registering for the REST API at "
+        "http://www.cortical.io/resources_apikey.html")
       raise Exception("Missing API key.")
 
-    self.apiKey  = os.environ['CEPT_API_KEY']
+    self.apiKey  = os.environ['REST_API_KEY']
 
-    self.client = pycept.Cept(self.apiKey, cacheDir=CACHE_DIR)
+    self.client = CorticalClient(self.apiKey, cacheDir=CACHE_DIR)  # cortipy object
```
Comment (subutai) [inline fluent/corti_py.py]: I think the default cacheDir should be explicit, as in `cacheDir='./cache'` and then get rid of the global variable CACHE_DIR which serves no other purpose.

File: fluent/corti_py.py [hunk 2]
```diff
@@ -18,30 +18,39 @@
...
+
+class CortiPy():
 
 
   def __init__(self):
-    if 'CEPT_API_KEY' not in os.environ:
-      print("Missing CEPT_API_KEY environment variable.")
-      print("You can retrieve this by registering for the CEPT API at ")
-      print("http://cept.github.io/CEPT-Website/developers_apikey.html")
+    if 'REST_API_KEY' not in os.environ:
```
Comment (subutai) [inline fluent/corti_py.py]: Should have a more specific name, like `CORTICAL_API_KEY`. REST_API is too generic.

File: setup.py [hunk 3]
```diff
@@ -22,9 +22,9 @@
 from setuptools import setup
 
 setup(name='Fluent',
-      version='0.1',
+      version='0.2',
       description=('A platform for building language / NLP-based '
-                   'applications using NuPIC and CEPT.'),
+                   'applications using NuPIC and Cortical.io REST.'),
```
Comment (subutai) [inline setup.py]: I think their company name cortical.io uses a lowercase c.

**Comment by subutai** [comment]
Nice - left some comments, otherwise :+1:

**Comment by BoltzmannBrain** [comment]
@subutai added (what should be) the last few changes for initially setting up nupic.fluent to run with cortipy. Once these merge I'll work more on the architecture we discussed yesterday.

**Pull request merged**